src/Server/Controllers/StatusUpdatesController.cs

```diff
@@ -37,4 +38,18 @@ public IEnumerable<StatusUpdate> All()
     {
         return _statusUpdates.FetchAll();
     }
+
+    /// <summary>
+    /// Creates a new status update.
+    /// </summary>
+    /// <param name="statusUpdate">The status update to add.</param>
+    /// <returns>If the status update was successfully added.</returns>
+    [HttpPut]
+    [Authorize("AdminOnly")]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    public IActionResult Create([FromBody] StatusUpdate statusUpdate)
+    {
+        _statusUpdates.Add(statusUpdate);
+        return Ok();
+    }
```

Comment:
You can rewrite simple controller action methods into an implicit form:
```suggestion
    public void Create([FromBody] StatusUpdate statusUpdate)
    {
        _statusUpdates.Add(statusUpdate);
    }
```
If this method doesn't throw, then it returns 200 OK.

You only really need to make it return `IActionResult` if you want to decide what status code is returned.